src/test/java/github/weichware10/util/DataPointTest.java

```diff
@@ -0,0 +1,75 @@
+package github.weichware10.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * Testet {@link DataPoint}.
+ */
+public class DataPointTest {
+    /**
+     * Testet, ob dataId richtig gesetzt wird.
+     */
+    @Test
+    public void dataIdShouldBeSetCorrect() {
+        int id1 = (int) Math.random() * 1000;
+        DataPoint dataPoint1 = new DataPoint(id1, 0, new int[]{0, 0}, new int[]{0, 0});
+        assertEquals("dataPoint1.dataId sollte " + id1 + " sein", id1, dataPoint1.dataId);
+
+        int id2 = (int) Math.random() * 1000;
+        DataPoint dataPoint2 = new DataPoint(id1, 0, new int[]{0, 0}, new int[]{0, 0}, 0.0f);
+        assertEquals("dataPoint2.dataId sollte " + id2 + " sein", id2, dataPoint2.dataId);
+    }
+
+    /**
+     * Testet, ob timeOffset richtig gesetzt wird.
+     */
+    @Test
+    public void timeOffShouldBeSetCorrect() {
+        float id1 = (float) Math.random() * 1000;
+        DataPoint dataPoint1 = new DataPoint(0, id1, new int[]{0, 0}, new int[]{0, 0});
+        assertEquals("dataPoint1.timeOffset sollte " + id1 + " sein",
+            id1, dataPoint1.timeOffset, 0.0001f);
+
+        float id2 = (float) Math.random() * 1000;
+        DataPoint dataPoint2 = new DataPoint(0, id2, new int[]{0, 0}, new int[]{0, 0}, 0.0f);
+        assertEquals("dataPoint2.timeOffset sollte " + id2 + " sein",
+            id2, dataPoint2.timeOffset, 0.0001f);
+    }
+
+    @Test
+    public void coordinatesShouldBeSetCorrect() {
+        int[] coordinates1 = new int[] {(int) Math.random() * 1000, (int) Math.random() * 1000};
+        DataPoint dataPoint1 = new DataPoint(0, 0, coordinates1, new int[]{0, 0});
+        assertTrue("dataPoint1.coordinates sollten {" + coordinates1[0] +
+            ", " + coordinates1[1] + "} sein", dataPoint1.coordinates.equals(coordinates1));
+
+        int[] coordinates2 = new int[] {(int) Math.random() * 1000, (int) Math.random() * 1000};
+        DataPoint dataPoint2 = new DataPoint(0, 0, coordinates2, new int[]{0, 0}, 0.0f);
+        assertTrue("dataPoint2.coordinates sollten {" + coordinates2[0] +
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.OperatorWrapCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'+' should be on a new line.